src/components/dashboard/FeedItems/FeedListItem.js

```diff
@@ -45,6 +45,7 @@ const getStylesFromProps = ({ theme }) => ({
     },
     elevation: 1,
     marginHorizontal: theme.sizes.default,
+    height: theme.feedItems.height,
     maxHeight: theme.feedItems.height,
```

Comment:
If we are setting a fixed height. shouldn't we remove max and min height? 
Actually having a fixed height sounds like it should be avoidable, in the feed item. Perhaps that fixed height should be a consequence of inner sizes.

Reply:
Yeah, that's there because the contents were overflowing the container on certain devices (and me trying to fix it).

I'm removing `minHeight` as it's redundant, but I think a fixed `height` is necessary / mandatory for this type of item, and I'm forcing a `maxHeight` equal to `height` to avoid content overflow.

It might have something to do with some `flexGrow: 1` or a similar property, but I don't want to delete that because I'm almost certain that other layout's behaviors depend on that. Code is a little touchy around there.